recipe/meta.yaml

```diff
@@ -19,23 +19,32 @@ outputs:
         - {{ pin_subpackage("ucx", max_pin="x.x") }}
       ignore_run_exports_from:
         - {{ compiler("cuda") }}
+        - cuda-cudart-dev          # [(cuda_compiler_version or "None").startswith("12")]
+      missing_dso_whitelist:
+        - "*libcuda.so*"
+        - "*libcudart.so*"
+        - "*libnvidia-ml.so*"
     requirements:
       build:
         - {{ compiler("c") }}
         - {{ compiler("cxx") }}
         - {{ compiler("cuda") }}
+        - libgomp
```

Comment:
We had discussed whether to use OpenMP before: https://github.com/conda-forge/ucx-split-feedstock/issues/119

From that we decided it was needed. So let's drop

```suggestion
```

Reply:
Looks like it is not possible to build without OpenMP atm due to a bug: https://github.com/openucx/ucx/issues/9848

Have gone ahead and added it back explicitly. Hopefully we can revisit in the future

Reply:
@jakirkham I added it because I saw a warning for `libgomp.so` was raised by conda-build during linking check, so yes it has to be added if we (I) want to be pedantic.